serializers/signupresponse.go

```diff
@@ -0,0 +1,72 @@
+package serializers
+
+import (
+	"app/models"
+	"time"
+)
+
+type UserResponse struct {
+	UserID             uint               `json:"user_id"`
+	Email              string             `json:"email"`
+	LastModified       string             `json:"last_modified"`
+	FirstName          string             `json:"first_name"`
+	LastName           string             `json:"last_name"`
+	DateOfBirth        string             `json:"date_of_birth"`
+	Gender             string             `json:"gender"`
+	MaritalStatus      string             `json:"marital_status"`
+	ResidentialDetails ResidentialDetails `json:"residential_details"`
+	OfficeDetails      OfficeDetails      `json:"office_details"`
+	Token              Token              `json:"token"`
+}
+
+type Token struct {
+	Key        string `json:"key"`
+	ExpiryTime string `json:"expiry_time"`
+}
+
+func BuildUserResponse(userDetails models.UserDetails, residentialDetails models.ResidentialDetails, officeDetails models.OfficeDetails, token string, expiryTime string) UserResponse {
```

Comment:
Using functions like these will become very difficult to use with larger responses